src/main/resources/db/changelog/v.0.1/db.changelog-coupons.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: 13-create-coupons-table
+      author: elizaveta_dubogryz
+      changes:
+        - createTable:
+            tableName: coupons
+            columns:
+              - column:
+                  name: id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+              - column:
+                  name: discount_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+              - column:
+                  name: date
+                  type: timestamp
```

Comment:
This way, the certain time of coupon  retrieving would be generated automatically at POST query in service layer.
Do I think the right way? Just this one it could be nullable: false

Reply:
I don't think so